src/Server/appsettings.Development.json

```diff
@@ -17,7 +17,17 @@
     },
     "services": {
       "platform": {
-        "endpoint": "192.168.20.209:7000"
+        "endpoint": "192.168.20.209:7000",
```

Comment:
Is this a real ip? Maybe `10.0.0.1` or something like that may be better

Reply:
This is for `Development` only. 😅